Sources/SwiftGodotMacroLibrary/SceneTreeMacro.swift

```diff
@@ -0,0 +1,97 @@
+// 
+//  SceneTreeMacro.swift
+//  SwiftGodot
+//
+//  Created by Marquis Kurt on 6/22/23.
+//
+
+import Foundation
+import SwiftCompilerPlugin
+import SwiftDiagnostics
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SwiftSyntaxMacros
+
+public struct SceneTreeMacro: AccessorMacro {
```

Comment:
I am still a bit concerned that we have duplicate functionality now, and would rather only have one way of doing things.

I am ok taking this over my existing solution, but we should at least make this default the path to being the name of the variable, so users do not have to repeat themselves, something like:

```swift
@SceneTree var myTimer: Timer
```

Should infer that the name of the timer in the scene is `myTimer`